lib/brain.js

```diff
@@ -682,6 +686,9 @@ Brain.prototype.scanPDF = function scanPDF () {
     }
 
     idCardData.setData(result)
+    if (_.get('idCardData')(this.trader.triggersAutomation) === MANUAL) {
+      this.manualTriggersDataProvided.idCardData = true
+    }
```

Comment:
You can refactor all the code related with this to a function I think

```suggestion
    function setManualTrigger (code) {
      if (_.get(code)(this.trader.triggersAutomation) === MANUAL) {
        this.manualTriggersDataProvided[code] = true
      }
    }
```